website/src/content/docs/docs/libraries/xml/guide.md

```diff
@@ -616,9 +629,11 @@ Author:
 ```tsp
 model Book {
   @Xml.attribute
-  id: string;
+  id: integer;
```

Comment:
The yaml example for this needs to be updated so that the `Book.properties.id.xml.attribute` field is `true`.